setup.py

```diff
@@ -379,6 +432,7 @@ def get_hdf5_version(headername):
 # is not a good idea.
 CFLAGS = os.environ.get('CFLAGS', '').split()
 LIBS = os.environ.get('LIBS', '').split()
+USE_PKGCONFIG = None
```

Comment:
It would not be better to unconditionally use pkg-config if it is found in the system?